ova/generate_ova.sh

```diff
@@ -109,16 +109,16 @@ build_ova() {
 
     vagrant destroy -f
 
-    tar -xvf ${OVA_VM}
+    tar -xvf "${OVA_VM}"
 
     echo "Setting up ova for VMware ESXi"
 
     # Configure OVA for import to VMWare ESXi
-    python Ova2Ovf.py -s ${OVA_VM} -d ${OVA_FIXED}
+    python3 Ova2Ovf.py -s "${OVA_VM}" -d "${OVA_FIXED}"
```

Comment:
Not all systems have Python 3 installed by default, this should be left as `python` to use the default python in the system.

Maybe we could add a conditional to use python or python3 if it exists.
```suggestion
    if [ -n "$(command -v python)" ]; then
        python Ova2Ovf.py -s "${OVA_VM}" -d "${OVA_FIXED}"
    elif [ -n "$(command -v python3)" ]; then
        python3 Ova2Ovf.py -s "${OVA_VM}" -d "${OVA_FIXED}"
    else
        echo "Cannot find python"
        clean 1
    fi
    
```